op-service/log/cli.go

```diff
@@ -147,9 +149,12 @@ func FormatHandler(ft FormatType, color bool) func(io.Writer) slog.Handler {
 		return log.NewTerminalHandler(w, color)
 	}
 	logfmtHandler := func(w io.Writer) slog.Handler { return log.LogfmtHandlerWithLevel(w, log.LevelTrace) }
+	logfmtMsHandler := func(w io.Writer) slog.Handler { return LogfmtMsHandlerWithLevel(w, log.LevelTrace) }
```

Comment:
I am including the `slog` handler in this repo, and not `op-geth`, so that we keep diff between upstream `geth` and `op-geth` minimal.

Reply:
If it's not too difficult I'd suggest it is worth submitting this new format to upstream geth as well.